packages/tpetra/core/src/Tpetra_Import_Util2.hpp

```diff
@@ -293,6 +323,17 @@ sortCrsEntries (const rowptr_array_type& CRS_rowptr,
     vals_array_type>::sortCrsEntries (CRS_rowptr, CRS_colind, CRS_vals);
 }
 
+template<typename rowptr_array_type, typename colind_array_type>
+void
+sortCrsEntries (const rowptr_array_type& CRS_rowptr,
+                const colind_array_type& CRS_colind)
+{
+  // Generate dummy values array
+  colind_array_type CRS_vals;
+  Impl::SortCrsEntries<rowptr_array_type, colind_array_type,
```

Comment:
How come you can't call the three-argument version of sortCrsEntries directly here?  Why do you need to refer to the implementation in Impl::SortCrsEntries?